build.gradle.kts

```diff
@@ -37,19 +34,22 @@ plugins {
     id("io.github.gradle-nexus.publish-plugin") version "2.0.0"
 }
 
+group = "org.eclipse.zenoh"
+version = file("version.txt").readText()
+
 nexusPublishing {
     repositories {
         sonatype {
+            nexusUrl = uri("https://s01.oss.sonatype.org/service/local/")
+            snapshotRepositoryUrl = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
I think we should update the URLs to use the legacy ones, per https://github.com/eclipse-zenoh/zenoh-kotlin/pull/331/files#diff-5869e8b2603421f6bcc39d64924dfeaf53c7d8b8c5a57cf6bba16a0ab9ed1bd4L163-L165